### ISSUE TITLE
Add version numbers to Notepad++ item Add/Remove Programs

### ISSUE THREAD

**Issue by SwarfegaGit** [notepad-plus-plus/notepad-plus-plus#3713]
### Description of the Issue
I use Desired State Configuration (DSC) to manage some Remote Desktop Servers to ensure they all provide the same user experience.  I use DSC to install Notepad++.  The issue is when there is a new version released DSC has no way to know if a new version needs to be installed.  Typically this is done via a product ID if the installer was an MSI file.  Since this is a flat EXE though it uses the items name that appears in Add/Remove Programs.  Since the name is always the same between versions DSC never considers the upgraded Notepad++ executables.  Is there any chance you can add the version number to the items name?

Example of Notepad++:

Notepad++ (64-bit x64)

Examples from other programs:

Mozilla Firefox 5.5.0.3 (x86 en-GB)
PuTTY release 0.70
WinSCP 5.11
7-Zip 16.04 (x64 edition)

### Steps to Reproduce the Issue
Not Applicable

### Expected Behavior
Not Applicable

### Actual Behavior
Not Applicable

### Debug Information
Not Applicable

**Comment by SinghRajenM** [comment]
Does it not work for you?

![image](https://user-images.githubusercontent.com/14791461/30173565-b8962832-9415-11e7-80bf-b2a389dca2c6.png)

**Comment by SwarfegaGit** [comment]
Yes. But DISC doesn't look at that. It uses the name.  If you look at the other apps in your list they include the version number in them.
